### PR TITLE
Markets Single Table

### PR DESCRIPTION
**Pull request by franm91**
image
image

### CODE REVIEW

File: components/markets/MarketsTables/poolTable/index.tsx [hunk 1]
```diff
@@ -148,47 +166,43 @@ const PoolTable: FC<PoolTableProps> = ({ isLoading, headers, rows, rateType }) =
...
-                        {rateType === 'fixed' && (
+                        <Grid container alignItems="center" gap={0.5}>
+                          {depositMaturity ? (
+                            <LockIcon sx={{ fontSize: 16, color: palette.operation.fixed }} />
+                          ) : (
+                            <ImportExportIcon sx={{ fontSize: 16, color: palette.operation.variable }} />
+                          )}
                           <Typography width="fit-content" variant="subtitle2" color="grey.500">
-                            {depositMaturity ? parseTimestamp(depositMaturity) : ''}
+                            {depositMaturity ? parseTimestamp(depositMaturity) : 'Open-endened'}
```
Comment (jgalat) [inline components/markets/MarketsTables/poolTable/index.tsx:197]: missing t

**Review by jgalat — DISMISSED** [review]
:rocket:

File: components/markets/MarketsTables/poolTable/index.tsx [hunk 2]
```diff
@@ -41,17 +42,25 @@ export type TableRow = {
...
   const { query } = useRouter();
   const { handleActionClick, isDisable } = useActionButton();
   const assets = useAssets();
+  const { palette } = useTheme();
   const defaultRows = useMemo<TableRow[]>(() => assets.map((s) => ({ symbol: s })), [assets]);
   const { setOrderBy, sortData, direction: sortDirection, isActive: sortActive } = useSorting<TableRow>();
   const tempRows = isLoading ? defaultRows : rows;
 
+  const getRateType = useCallback((maturity: bigint | undefined) => {
+    if (maturity === undefined) return 'floating';
```
Comment (sebipap) [inline components/markets/MarketsTables/poolTable/index.tsx:60]: minor: it could be
```suggestion
    return !!maturity ? 'fixed' : 'floating'
```

**Review by sebipap — DISMISSED** [review]
code LGTM

**Pull request merged**